core/services/ocrcommon/telemetry.go

```diff
@@ -1,64 +1,137 @@
 package ocrcommon
 
 import (
+	"context"
 	"encoding/json"
 
+	"github.com/ethereum/go-ethereum/common"
```

Comment:
Is it ok for geth types to be in here? Or should this package be chain-agnostic?

Reply:
I guess it's ok, the only thing used from this is `BytesToHash()` and I see that mercury also uses geth